bucket/blender-launcher.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.14.1",
+    "description": "Standalone client for managing official builds of Blender 3D",
+    "homepage": "https://github.com/DotBow/Blender-Launcher",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/DotBow/Blender-Launcher/releases/download/v1.14.1/Blender_Launcher_v1.14.1_Windows_x64.zip",
+            "hash": "6b64bf298e66405f2e04241ba9372016cd3361f9439dc2b8e27b61f9dbc69474"
+        }
+    },
+    "bin": "Blender Launcher.exe",
```

Comment:
```suggestion
```

Reply:
Blender Launcher accepts the following command line arguments.

    -help                        * Show command line arguments sheet
    -update                      * Run updater instead of the main application
    -debug                       * Set logging level to DEBUG
    -set-library-folder "%path%" * Set library folder
    -offline                     * Disable scraper thread

https://github.com/DotBow/Blender-Launcher/blob/v1.14.0-stable/source/main.py#L35-L44